### DISCUSSION TITLE
辅助函数全都得放到debugtalk.py,导致其非常臃肿怎么破

### DISCUSSION THREAD

**Discussion by DishuiH** [httprunner/httprunner#1710] [2 - 答疑（Q&A）]
debugtalk.py只允许放在根目录下，辅助函数全都得放到debugtalk.py,导致其非常臃肿怎么办？

**Comment by s-panjy** [comment]
同感，有没有可能支持自行拆分多个py，用debugtalk引入

**Comment by WhereTo** [comment]
把实现都放在其他文件夹 debugtalk.py只保留函数定义
